client/mass/store.js

```diff
@@ -595,6 +595,8 @@ function validatePlotTerm(t, vocabApi) {
 			break
 		case TermTypes.SINGLECELL_GENE_EXPRESSION:
 			break
+		case TermTypes.CELLTYPE:
```

Comment:
name is too general. please rename it SINGLECELL_CELLTYPE so it's specific to sc data